docs/docs/en/guide/resource/udf-manage.md

```diff
@@ -0,0 +1,45 @@
+# UDF Manage
+
+> The resource management and file management functions are similar. The difference is that the resource management is the UDF upload function, and the file management uploads the user programs, scripts and configuration files.
```

Comment:
I think we should avoid using syntax like `>` in the in paragraph, unless some content like **NOTE**, same as the next sentence
```suggestion
The resource management and file management functions are similar. The difference is that the resource management is the UDF upload function, and the file management uploads the user programs, scripts and configuration files.
```

Reply:
> 

Sounds good! I will update it later.